app/routers/learning_progress.py

```
@@ -0,0 +1,31 @@
+from app.models.learning_progress import LearningStatus
+from app.schemas.learning_progress_schema import LearningProgressResponse
+from app.services.learning_progress_service import LearningProgressService
+
+router = APIRouter()
+
+
+@router.get("/users/{user_id}/quizzes", response_model=List[LearningProgressResponse])
+async def get_learning_progress(
+    user_id: int,
+    status: Optional[LearningStatus] = None,
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)